control_panel_api/models.py

```diff
@@ -26,7 +29,7 @@ def get_short_name(self):
 
     @property
     def aws_role_name(self):
-        return f"{settings.ENV}_user_{self.username.lower()}"
+        return f"{settings.ENV}_user_{self.auth0_id}"
```

Comment:
This shouldn't change, I'm not even sure you can call an IAM role something like `dev_user_github|123`, plus it would break compatibility with existing stuff.

Reply:
Ok. Ticket mentions it. Will revert.